samples/ballerina-by-example/examples/kafka_message_producer_transaction_advanced/kafka_message_producer_transaction_advanced.bal

```diff
@@ -39,8 +39,8 @@ service<kafka:Consumer> kafkaService bind consumer {
 
     onMessage(kafka:ConsumerAction consumerAction, kafka:ConsumerRecord[] records) {
         // Dispatched set of Kafka records to service, We process each one by one.
-        foreach record in records {
-            processKafkaRecord(record);
+        foreach entry in records {
```

Comment:
I guess we no longer can use record since it s a keyword. Can't we find more suitable name for this?
Eg:- foreach entry in records    
May be we can use kafkaRecord over entry, since record is terminology that comes from Kafka API.